src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -216,7 +224,18 @@ public string Version
         /// <summary>
         ///     The maximum amount of tesselation divisions used for geometry visualization.
         /// </summary>
-        public int MaxTesselationDivisions { get; set; }
+        public int MaxTessellationDivisions
+        {
+            get { return maxTessellationDivisions; }
+            set
+            {
+                maxTessellationDivisions = value;
+                if (RenderPackageFactory != null)
+                {
+                    RenderPackageFactory.MaxTessellationDivisions = maxTessellationDivisions;
+                }
+            }
+        }
 
```

Comment:
Honestly, if it were up to me I'd be removing this field from `DynamoModel`, not using it more.  Plus, haven't you added a duplicate field on `IRenderPackageFactory`?